core/src/datayoga_core/result.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import Optional
+
+Status = Enum("Status", "SUCCESS REJECTED FILTERED")
+
+
+class Result():
+    def __init__(self, status: Status, message: Optional[str] = None):
+        self.status = status
+        self.message = message
+
+    def __eq__(self, other):
+        return (
+            isinstance(other, self.__class__) and
+            self.status == other.status and
+            self.message == other.message)
+
+    @staticmethod
+    def success() -> Result:
```

Comment:
why a static method and not a contstant/enum?

Reply:
answered in a different comment